### ISSUE TITLE
RemoveSplatScene is never done if there is only one scene.

### ISSUE THREAD

**Issue by fred-ais-pl** [mkkellogg/GaussianSplats3D#254]
I am using version `0.4.2` with `sharedMemoryForWorkers: false`, `progressiveLoad: true` options. If I call `removeSplatScene` on the only scene, the viewer will freeze and **Removing splat scene...** message never disappears. Looks like [addSplatBuffers](https://github.com/mkkellogg/GaussianSplats3D/blob/main/src/Viewer.js#L1035C5-L1035C20) is never resolved because there is no more message from `sortWorker`.

Here is snippet code to reproduce the issue
```
viewer.addSplatScene(url, { 'progressiveLoad': true })
    .then(() => new Promise(resolve => setTimeout(resolve, 5000))) // wait for scene to load
    .then((() => viewer.removeSplatScene(0)))
```

`removeSplatScene` works as expected if there are at least two scenes, but if I want progressiveLoad only one is allowed.

**Comment by mkkellogg** [comment]
Ah, thanks for pointing this out! I'll try to get a fix for this into the next release. In the meantime a work-around would be to just to dispose and then re-create the viewer when there is only one scene (I know it's a bit of a pain to do it that way).

**Comment by fred-ais-pl** [comment]
Thank you, do you know when I can expect the next release? Is it a matter of a week or a month?

**Comment by mkkellogg** [comment]
The next release is at least another couple of weeks away, possibly up to a month away (I don't have a ton of time to dedicate to development right now).

**Comment by mkkellogg** [comment]
I made an update in the `2dgs` branch that should address this issue, would you be willing to see if it fixes it for you?

**Comment by fred-ais-pl** [comment]
> I made an update in the `2dgs` branch that should address this issue, would you be willing to see if it fixes it for you?

I confirm it works now as expected.

**Comment by mkkellogg** [comment]
Quick update: this fix is now in `main` and [Release v0.4.3](https://github.com/mkkellogg/GaussianSplats3D/releases/tag/v0.4.3)
